### ISSUE TITLE
about importing other pre-trained model

### ISSUE THREAD

**Issue by tomtomtong** [Syn-McJ/TFClassify-Unity#2]
Hello. How do you convert the pre-trained model into .bytes format?
Do you use freeze_graph.py? Thx

**Comment by Syn-McJ** [comment]
Hi @tomtomtong,

No, I did not use `freeze_graph`, I just took the model from [tensorflow repo](https://github.com/tensorflow/tensorflow) and simply renamed it so that it has .bytes extension instead of .pb. Same thing for some MobileNet models I tried, for example the one I got flowing [this article](https://hackernoon.com/creating-insanely-fast-image-classifiers-with-mobilenet-in-tensorflow-f030ce0a2991). Simply switch extension to .bytes and use it as you would use it in the android example from tensorflow repo.
